web/404.html

```diff
@@ -1,63 +1,96 @@
-<html>
+<!DOCTYPE html>
+<html lang="en">
 
 <head>
   <title>IATI Bulk Data Service</title>
-  <link href="https://cdn.jsdelivr.net/npm/iati-design-system@1.2.0/dist/css/iati.min.css" rel="stylesheet" />
+  <link href="https://cdn.jsdelivr.net/npm/iati-design-system@4.0.0/dist/css/iati.min.css" rel="stylesheet" />
 </head>
 
-<body>
+<body class="iati-design-system--enabled">
   <div class="iati-title-bar">
     <header class="iati-header">
-      <div>
-        <a href="https://iatistandard.org/">
-          <img src="http://styles.iatistandard.org/assets/svg/source/logo-colour.svg" class="iati-header__logo" />
-        </a>
+
+      <div class="iati-header__section iati-header__section--first">
+        <div class="iati-header__container">
+
+          <a href="https://iatistandard.org/" aria-label="Go to IATI homepage">
+            <img class="iati-header__logo" alt=""
+              src="http://styles.iatistandard.org/assets/svg/source/logo-colour.svg" />
+          </a>
+
+          <nav class="iati-header__general-nav">
+            <ul class="iati-piped-list">
+              <li><a href="https://iatistandard.org/en/about/">About IATI</a></li>
+              <li><a href="https://iatistandard.org/en/using-data/">Use Data</a></li>
+              <li><a href="https://iatistandard.org/en/guidance/publishing-data/">Publish Data</a></li>
+              <li><a href="https://iatistandard.org/guidance/get-support/">Contact</a></li>
+            </ul>
+          </nav>
+        </div>
+      </div>
+
+      <div class="iati-header__section iati-header__section--last iati-brand-background">
+        <div class="iati-header__container iati-brand-background__content">
+
+          <div class="iati-header__actions">
+            <div class="iati-country-switcher">
+              <label for="iati-country-switcher" class="iati-country-switcher__label">Choose your language</label>
+              <select id="iati-country-switcher" class="iati-country-switcher__control">
+                <option>English</option>
+              </select>
+            </div>
+          </div>
+
+          <div class="iati-header-title">
+            <p class="iati-header-title__heading">IATI Bulk Data Download</p>
```

Comment:
Is it possible to make this match the standard "IATI Tools" header, as seen here: https://tables.iatistandard.org and https://iati.github.io/design-system/?path=/docs/layout-page--docs?  I think that's now a requirement of the design system. So it should have the two-level header, with "IATI Tools", and then under that it says which tool.